From 34c2c4008c78349f2ed9c62f367771c0bc80d435 Mon Sep 17 00:00:00 2001
Message-Id: <34c2c4008c78349f2ed9c62f367771c0bc80d435@dist-git>
From: Martin Kletzander <mkletzan@redhat.com>
Date: Mon, 7 Apr 2014 10:25:07 +0200
Subject: [PATCH] virsh: Fix heading in manpage

'=item' needs to follow an empty line.

https://bugzilla.redhat.com/show_bug.cgi?id=996840

RHEL-only (fixed upstream with addition of new flags in commit
652a2ec63051f1c4570f49858db445b686abbb0e not worth rebasing)

Signed-off-by: Martin Kletzander <mkletzan@redhat.com>
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 tools/virsh.pod | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/virsh.pod b/tools/virsh.pod
index e010d8d..806f247 100644
--- a/tools/virsh.pod
+++ b/tools/virsh.pod
@@ -1955,6 +1955,7 @@ separated by comma, e.g. --cap pci,scsi, valid capability types include
 'scsi', 'storage'. If I<--tree> is used, the output is formatted in a tree
 representing parents of each node. I<cap> and I<--tree> are mutually
 exclusive.
+
 =item B<nodedev-reattach> I<nodedev>
 
 Declare that I<nodedev> is no longer in use by any guests, and that
-- 
1.9.1

